extensions/parquet/table/src/main/java/io/deephaven/parquet/table/transfer/TransferObject.java

```diff
@@ -30,135 +29,164 @@
  */
 public interface TransferObject<B> extends SafeCloseable {
     static <DATA_TYPE> TransferObject<?> create(
-            @NotNull final Map<String, Map<ParquetCacheTags, Object>> computedCache,
             @NotNull final RowSet tableRowSet,
-            @NotNull final ColumnSource<DATA_TYPE> columnSource,
-            @NotNull final ColumnDefinition<DATA_TYPE> columnDefinition,
-            @NotNull final ParquetInstructions instructions) {
+            @NotNull final ParquetInstructions instructions,
+            @NotNull final Map<String, Map<ParquetCacheTags, Object>> computedCache,
+            @NotNull final String columnName,
+            @NotNull final ColumnSource<DATA_TYPE> columnSource) {
         Class<DATA_TYPE> columnType = columnSource.getType();
         if (int.class.equals(columnType)) {
```

Comment:
You don't need to use `.equals` when comparing class objects. Reference equality (`==`) is sufficient.